### ISSUE TITLE
Add support for Arms Warrior

### ISSUE THREAD

**Issue by MartijnHols** [WoWAnalyzer/WoWAnalyzer#2405]
Since I do not know Arms Warrior, I can not specify a definition of done. If you think it's ready to be marked as supported, you'll have to somehow convince us of this as well.

**Comment by MartijnHols** [comment]
[![Bountysource](https://api.bountysource.com/badge/issue?issue_id=64407761)](https://www.bountysource.com/issues/64407761-add-support-for-arms-warrior)

**Comment by RyanFletcher86** [comment]
I'd like to start looking into this. First step is clearly creating a definition of done. Since I haven't played world of warcraft in years (and I don't intend to resume), I'll start by finding relevant guides on how to play the Arms Warrior.

I'll also need to gain an understanding of the metrics that are expected. When my research is done I'll make another post defining the scope of what I intend to implement. Once somebody with authority confirms that my scope would be acceptable then and only then will I begin writing code.

If anybody has external resources (existing functionality in the code base to reference, how to play arms warrior guides, write ups of how to do the various calculations etc), that would be useful to my research :).

**Comment by MartijnHols** [comment]
Updated bounty to equal a 12 month subscription to the game:

[![Bountysource](https://api.bountysource.com/badge/issue?issue_id=64407761)](https://www.bountysource.com/issues/64407761-add-support-for-arms-warrior)

**Comment by torothin** [comment]
I am currently working on a doc to express the requirements for completeness.  It will be at the following link:
https://docs.google.com/document/d/1v6OmuUTdAZuY9AZeajh4nsVmnCR0bwX0Zx3My_eE5VQ/edit?usp=sharing
[https://docs.google.com/document/d/1v6OmuUTdAZuY9AZeajh4nsVmnCR0bwX0Zx3My_eE5VQ/edit?usp=sharing](url)

Once I feel the doc is complete I will begin checking what is already done and investigate whether or not creating a project in Github is worthwhile.

**Comment by Karg3ths** [comment]
I've been looking into the doc. And I've found few things which should be updated due to the Shadowlands changes. For example using Mortal Strike in the Execute phase is essential, because Execute no longer refreshes Deep Wounds on the target. The playstyle and optimal talents are based on the selected covenant and legendaries. Venthyr having second execute phase when target is above 80% of it's HP. Night Fey and Kyrian having their AOE abilities used properly (Usually on multiple targets so more rage will be generated).

**Comment by Chasson1992** [comment]
> I've been looking into the doc. And I've found few things which should be updated due to the Shadowlands changes. For example using Mortal Strike in the Execute phase is essential, because Execute no longer refreshes Deep Wounds on the target. The playstyle and optimal talents are based on the selected covenant and legendaries. Venthyr having second execute phase when target is above 80% of it's HP. Night Fey and Kyrian having their AOE abilities used properly (Usually on multiple targets so more rage will be generated).

I’ve started to put together the basics of updating the Arms Warr page by adding in shadowlands capabilities for covenants, conduits and the like.

I’m an Arms Warr main, have been since classic, so I understand the rotation pretty well. That google doc definitely doesn’t reflect the latest ideal rotations. I’m going to create things based off my knowledge of the game for now. If anyone has any specifics to include for arms warrior please reach out to me!
